ui/widgets/datepicker.js

```diff
@@ -79,30 +81,33 @@ var widget = $.widget( "ui.datepicker", {
 	_getCreateOptions: function() {
 		var max = this.element.attr( "max" ),
 			min = this.element.attr( "min" ),
+			parser = new Globalize( "en" ).dateParser( { raw: "yyyy-MM-dd" } ),
```

Comment:
This particular line is about parsing the input min / max attributes. So this pattern (`yyyy-MM-dd`) should never change, regardless of the used locale.

Anyway, if we want to keep it this way, we will need to tell users to add it, like done here (for multiple locales):  https://github.com/jquery/jquery-ui/pull/1755/commits/ea986f2808684c0112d56776d76ea35e9cdb77da

Using a simple split seems interesting as we could remove one of two (direct, not within Calendar widget) usages of Globalize in datepicker. In addition we could revert additions made above and keep the dependencies to a minimum.

So, +1 from me for using a simple split.